### ISSUE TITLE
Should exclude Javadoc

### ISSUE THREAD

**Issue by sebbASF** [Humbedooh/clc#17]
Javadoc is generated, so no point flagging it

**Comment by julianhyde** [comment]
Better, it should be configurable whether to scan javadoc. Maybe an 'ignore path' config parameter.

**Comment by sebbASF** [comment]
Not sure why one would ever want to scan Javadoc as well as the source from which it is generated.
There is already a way to ignore paths, but it means every project with Javadoc has to update their config.

**Comment by julianhyde** [comment]
Agreed, virtually no one wants to scan javadoc. But neither do you want the tool skipping directories because it wrongly thinks they contain javadoc.

**Comment by sebbASF** [comment]
The tool should anyway list which directories are skipped.

Note that Javadoc directory trees have a standard set of files, so should be reasonably easy to detect with few false positives
